test/zipObj.js

```diff
@@ -11,9 +11,8 @@ describe('zipObj', function() {
     eq(R.zipObj(['a', 'b', 'c'], [1, 2, 3, 4, 5, 6, 7]), {a: 1, b: 2, c: 3});
   });
 
-  it('extra keys are undefined', function() {
```

Comment:
@CrossEye noticed this test here, which implies that this functionality was intended. Just want to point it out in case I'm jumping the gun.